### PR TITLE
create swap1108.md

### PR DESCRIPTION
**Pull request by swap1108**
Please review this!

### CODE REVIEW

**Comment by palash25** [comment]
Thanks @swap1108 we will review it and let you know after the merge

@knrt10 I checked the travis log. I think it needs a rebuild and the tests might pass but I need your opinion on it.
Also there is an npm warning about a version bump for a dependency might wanna check that out too.

File: _contributors-app/swap1108.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+project:
+ - |
+   - [Algokills](https://www.algoskills.com): A website that provides an interactive platform for computer algorithms. A step taken to learn and contribute
+
+ - |
+    - [Differential Evolution](): Modification of Ranking based mutation in Differential Evolution (present)
+
+ - |
+    - [Activity recognition using smartphone sensors with auto-annotation in IoT](https://drive.google.com/file/d/0B3iCiFcNmJYseTFNZm1OVlJuV2c/view): An innovative way for human acitvity recognition.
```
[the remark this answers is not in the document]
Comment (palash25) [reply _contributors-app/swap1108.md]: Yeah Yeah Makman style review :clap:

**Comment by swap1108** [comment]
@palash25 @knrt10  Review This.

**Comment by swap1108** [comment]
@knrt10  and @palash25  Thanks for assistance, I seemed to have learned most of the open sourcing in this process.. LOL

**Pull request merged**